tests/foreman/longrun/test_provisioning_computeresource.py

```diff
@@ -23,88 +22,101 @@
 from robottelo.config import settings
 from robottelo.constants import FOREMAN_PROVIDERS
 from robottelo.constants import VMWARE_CONSTANTS
-from robottelo.decorators import skip_if_not_set
 from robottelo.helpers import host_provisioning_check
-from robottelo.helpers import ProvisioningCheckError
-from robottelo.test import CLITestCase
 from robottelo.utils.issue_handlers import is_open
 
 
-class ComputeResourceHostTestCase(CLITestCase):
-    """RHEVComputeResource CLI tests."""
+@pytest.fixture(scope="module")
+def bridge():
```

Comment:
This is overkill usage of fixtures, no need to wrap access to `settings` into fixtures.